--- origsrc/arora/src/src.pri	2008-07-15 01:55:58.000000000 -0500
+++ src/arora/src/src.pri	2008-07-15 02:03:08.656250000 -0500
@@ -14,8 +14,8 @@
 
 QT += webkit network
 
-GITVERSION=$$system(git log -n1 --pretty=format:%h)
-GITCHANGENUMBER=$$system(git log --pretty=format:%h | wc -l)
+GITVERSION=e4fb8d9
+GITCHANGENUMBER=251
 DEFINES += GITVERSION=\"\\\"$$GITVERSION\\\"\"
 DEFINES += GITCHANGENUMBER=\"\\\"$$GITCHANGENUMBER\\\"\"
 
@@ -104,7 +104,7 @@
 
 unix {
     isEmpty(PREFIX){
-        PREFIX = /usr/local
+        PREFIX = /usr
     }
 
     BINDIR = $$PREFIX/bin
